Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3.11.6-alpine3.18
+
+ENV PYTHONDONTWRITEBYTECODE=1
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+COPY requirements.txt .
```

Comment:
Consider copying only the necessary files for installing dependencies before running `pip install`. This can help leverage Docker's caching mechanism more effectively. For example, copy only `requirements.txt` first, run `pip install`, and then copy the rest of the application files.